easybuild/easyconfigs/p/PyZMQ/PyZMQ-18.1.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'PyZMQ'
+version = '18.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.zeromq.org/bindings:python'
+description = """Python bindings for ZeroMQ"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
@branfosj why not move this down to `GCCcore` toolchain?

Reply:
I've moved it.